iamserver/iamserver_test.go

```diff
@@ -711,7 +711,7 @@ func TestUsersChanged(t *testing.T) {
  * Private
  ******************************************************************************/
 
-func newTestClient(url string) (client *testClient, err error) {
+func newTestClient(url string) (client *testClient, err error) { // nolint:unparam
```

Comment:
What is wrong here?

Reply:
newTestClient` - `url` always receives `serverURL` (`"localhost:8088"`) (unparam)
addede commnet // param added for future porpoises